### PR TITLE
fix(e2e): fix e2e for v2

### PR DESCRIPTION
**Pull request by fullstackninja864**
#### What kind of PR is this?:

/kind fix

#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [ ] No console errors on web
- [ ] Tested on Light mode and Dark mode*
- [ ] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [ ] Added e2e tests*
- [ ] Added translations*

### CODE REVIEW

File: mobile-app/app/screens/AppNavigator/screens/Dex/components/PoolPairCards/PoolPairCards.tsx [hunk 1]
```diff
@@ -254,34 +255,36 @@ const PoolCard = ({
...
-    onAdd={() => onAdd(mappedPair, yourPair as WalletToken)}
-    onRemove={() => onRemove(mappedPair, yourPair as WalletToken)}
-    walletTokenAmount={new BigNumber((yourPair as WalletToken).amount)}
-    walletTokenPrice={getTokenPrice(
-      yourPair.symbol,
-      new BigNumber((yourPair as WalletToken).amount),
-      true
-      )}
-  />)}
+      <View testID={`pool_pair_row_${index}_${mappedPair.displaySymbol}`}>
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Dex/components/PoolPairCards/PoolPairCards.tsx:258]: curious why we added index here?

File: mobile-app/cypress/e2e/functional/wallet/dex/add_liquidity.spec.ts [hunk 2]
```diff
@@ -54,7 +56,7 @@ function percentageAmountButton (percentage: string, tokenAAmount: string, token
   cy.getByTestID('lp_tokens_to_receive_value_rhsUsdAmount').contains(lpTokenUsd)
 }
 
-context('Wallet - DEX - Add Liquidity', () => {
+context.only('Wallet - DEX - Add Liquidity', () => {
```
Comment (kyleleow) [inline mobile-app/cypress/e2e/functional/wallet/dex/add_liquidity.spec.ts:59]:
```suggestion
context('Wallet - DEX - Add Liquidity', () => {
```

File: mobile-app/cypress/e2e/functional/wallet/dex/dex_features.spec.ts [hunk 3]
```diff
@@ -169,14 +169,16 @@ context('Wallet - DEX - Features', () => {
...
+    cy.getByTestID('poolpair_token_details_add_liquidity').click()
     cy.url().should('include', 'DEX/AddLiquidity')
     cy.go('back')
-    cy.getByTestID(`pool_pair_swap-horiz_${poolpairDisplaySymbol}`).click()
+    cy.getByTestID('poolpair_token_details_composite_swap').click()
     cy.url().should('include', 'DEX/CompositeSwap')
     cy.getByTestID('token_select_button_FROM').should('have.attr', 'aria-disabled')
     cy.getByTestID('token_select_button_TO').should('have.attr', 'aria-disabled')
     cy.go('back')
+    cy.go('back')
```
Comment (kyleleow) [inline mobile-app/cypress/e2e/functional/wallet/dex/dex_features.spec.ts:181]: go back twice?

Comment (fullstackninja864) [reply mobile-app/cypress/e2e/functional/wallet/dex/dex_features.spec.ts:181]: yes its required here

**Pull request merged**